### PR TITLE
Gives the CE a pair of advanced scanner goggles.

### PR DESCRIPTION
**Pull request by Jackal-boop**
## About The Pull Request
makes a new type of goggle which combine the abilitiess of the t-ray scanners and the engine scannner goggles
## Why It's Good For The Game

Gives the CE some more defining gear that sets him apart from a standard engineer, helps the CE diagnose atmospheric issues without carrying a pair of thermal imaging goggles in his bag.

## Changelog
:cl:
add: Advanced scanner goggles for the Chief Engineer
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/engine_goggles.dm [hunk 1]
```diff
@@ -139,7 +139,7 @@
 	name = "optical t-ray scanner"
 	icon_state = "trayson-t-ray"
 	inhand_icon_state = "trayson-t-ray"
-	desc = "Used by engineering staff to see underfloor objects such as cables and pipes."
+	desc = "Goggles utilized by atmospheric technitians that support a suite of features"
```
Comment (Fikou) [inline code/modules/clothing/glasses/engine_goggles.dm:142]:
```suggestion
	desc = "Goggles utilized by atmospheric technicians that support a suite of features."
```
cmon bro if youre editing it do it good

File: code/modules/clothing/glasses/engine_goggles.dm [hunk 2]
```diff
@@ -203,6 +203,16 @@
...
 		flick_overlay_global(pic, list(viewer.client), duration)
 
+/obj/item/clothing/glasses/meson/engine/advanced //special goggles for the CE that have the abilities of the T-ray scanners and engineering scanners
+	name = "advanced scanner goggles"
+	icon_state = "trayson-adv"
+	inhand_icon_state = "trayson-adv"
+	desc = "A pair of advanced goggles combining the capabilities of both the engineering scanner goggles and t-ray scanner goggles"
+	range = 3
+
+
```
Comment (Fikou) [inline code/modules/clothing/glasses/engine_goggles.dm:213]:
```suggestion
	desc = "A pair of advanced goggles combining the capabilities of both the engineering scanner goggles and t-ray scanner goggles."
	range = 3
```

**Comment by optimumtact** [comment]
we do not provide heads of staff with extra loot to make them standout these days :-1:

**Comment by castawaynont** [comment]
the last thing the ce needs is more gamer loot that's just engineer+

**Comment by Jackal-boop** [comment]
> we do not provide heads of staff with extra loot to make them standout these days 👎

can i add them in the engie lathe after ressearch

**Comment by optimumtact** [comment]
That's the same node you get the normal scanner goggles mate

**Comment by Jackal-boop** [comment]
> That's the same node you get the normal scanner goggles mate

Tumblr_l_781717269417630

**Pull request closed without merging**